scripts/migration-task-script.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+import re
+import subprocess
+import sys
+
+command_to_run = "flask db upgrade"
```

Comment:
This matches up with what runs under https://github.com/communitiesuk/funding-service-design-application-store/blob/main/scripts/run_migrations_paas.py#L16